services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/internal/update/UpdateExpressionUtils.java

```diff
@@ -99,7 +103,7 @@ private static List<RemoveAction> removeActionsFor(Map<String, AttributeValue> a
     private static RemoveAction remove(String attributeName) {
         return RemoveAction.builder()
                            .path(keyRef(attributeName))
-                           .expressionNames(Collections.singletonMap(keyRef(attributeName), attributeName))
+                           .expressionNames(expressionNamesFor(attributeName))
```

Comment:
Do we have  test case that covers `REMOVE`  action ?
What is the scenario where "REMOVE" comes in to play

Reply:
Yeah, remove does not come into play here. I modified it to make it neater and reuse the expressionNamesFor method. I can undo these changes to make it less confusing in the commit log.